index.js

```diff
@@ -124,14 +125,16 @@ const runPackageManager = (params, buildType, options) => {
     const spinner = new Spinner();
     // Determine whether skin.json was amongst the chosen options
     const skinIncluded = (options.skin || []).some(pluginId => pluginId === 'skin-json');
+    // From 4.10.0 onwards, the skin.json is bundled inside the html5-skin plugin
+    const skinIsBundled = semver.satisfies(params.version, '>=4.10.0');
     // Filter plugin assets and obtain a list all required files and dependencies
     const resources = resourceManager.filterPluginResources(plugins, params.version, buildType, options);
 
     fs.emptyDirSync(config.BUILD_PATH);
 
     resourceManager.downloadResources(resources, mainBuildPath, true)
     .then(() => {
-      if (!skinIncluded) {
+      if (!skinIncluded || skinIsBundled) {
```

Comment:
this sounds reasonable. @aldigun do you remember why we chose absolute paths over relative?

customers can still change to absolute if needed but our default should be relative.